packages/web/src/components/track/desktop/TrackTile.tsx

```diff
@@ -106,8 +113,51 @@ const TrackTile = memo(
     const { isEnabled: isGatedContentEnabled } = useFlag(
       FeatureFlags.GATED_CONTENT_ENABLED
     )
+    const { isEnabled: isNewPodcastControlsEnabled } = useFlag(
+      FeatureFlags.PODCAST_CONTROL_UPDATES_ENABLED
+    )
+    const trackPositionInfo = useSelector((state: CommonState) =>
+      getTrackPosition(state, { trackId })
+    )
 
     const hasOrdering = order !== undefined
+    const isPodcast = genre === Genre.PODCASTS
+
+    const getDurationText = () => {
```

Comment:
same potential idea here to add components, so these large components dont get too crazy with dependinces, hooks, and render fn size